mm2src/mm2_main/src/lp_native_dex.rs

```diff
@@ -419,10 +418,6 @@ pub async fn lp_init_continue(ctx: MmArc) -> MmInitResult<()> {
     #[cfg(target_arch = "wasm32")]
     init_wasm_event_streaming(&ctx);
 
-    // This function spawns related WalletConnect related tasks needed for initialization before
-    // WalletConnect can be usable in KDF.
-    WalletConnectCtx::from_ctx(&ctx).mm_err(|err| MmInitError::WalletInitError(err.to_string()))?;
```

Comment:
I wonder why this was put here, @borngraced can you please provide your input?

Reply:
here are components that needs to be activated/initialized when creating WalletConnect session for the first time when KDF processing is running. Possible rpc timeout response if the init function takes more time than expected..
https://github.com/KomodoPlatform/komodo-defi-framework/blob/37bf06a76aa3b0f36f0ec68b6155de34115bab67/mm2src/kdf_walletconnect/src/lib.rs#L169-L202

Reply:
How long could this take? It's pretty fast on my machine but I am running on high-end hardware with high-end network infra so my tests shouldn't be in this calculation. If it really is just a couple of seconds, doing it in the first RPC invocation seems totally fine to me.

That said, the main goal of this PR is to avoid this extra runtime work for anyone who isn't using WalletConnect. There are plenty of other KDF use cases and gaining a few seconds (which will happen only once) off some RPCs doesn't justify spawning these functions for every single KDF instance IMO.